cs_tools/api/models/metadata.py

```diff
@@ -368,3 +370,52 @@ def details(
                 }
             )
         return r
+
+    @validate_arguments
+    def tml_export(
+        self,
+        export_ids: List[GUID],
+        format_type: TMLType = TMLType.yaml,
```

Comment:
can we rename to `formattype` to match rest api v1

Reply:
Renames to formattype.  This was a bug in the call that wasn't noticed because the default is TML (YAML).